webapp/channels/src/components/root/root.tsx

```diff
@@ -280,6 +281,7 @@ export default class Root extends React.PureComponent<Props, State> {
 
         if (prevState.shouldMountAppRoutes === false && this.state.shouldMountAppRoutes === true) {
             if (!doesRouteBelongToTeamControllerRoutes(this.props.location.pathname)) {
+                DesktopApp.reactAppInitialized();
```

Comment:
So desktop timer still exists or do we stop that as well. a bit tangent what do you feel about having loading in only webapp?

Reply:
ah i see. another reason is also the dark/light mode of screen loading indicator which is there in the desktop app based on users system theme